webblog/frontend/src/components/blog/BlogList.js

```diff
@@ -0,0 +1,81 @@
+import React, {Component} from 'react';
+import {connect} from 'react-redux';
+import PropTypes from 'prop-types';
+import {deleteBlog, getBlogList} from "../../actions/blogList";
+
+export class BlogList extends Component {
+
+    static propTypes = {
+        blogList: PropTypes.array.isRequired,
+        getBlogList: PropTypes.func.isRequired,
+        deleteBlog: PropTypes.func.isRequired
+    };
+
+    state = {
+        currentPage: 1,
+        blogsPerPage: 3
+    };
+
+    componentDidMount() {
+        this.props.getBlogList();
+    }
+
+    handleClick = (event) => {
+        this.setState({
+            currentPage: Number(event.target.id)
+        });
+    };
+
+    render() {
+        const {currentPage, blogsPerPage} = this.state;
+
+        const indexOfLastTodo = currentPage * blogsPerPage;
+        const indexOfFirstTodo = indexOfLastTodo - blogsPerPage;
+        const currentBlogList = this.props.blogList.slice(indexOfFirstTodo, indexOfLastTodo);
```

Comment:
is there any other method to find current Blog list ?

Reply:
this mechanism is used to show only that part of the blog list which is in the page.